File: 05_Demo/demo/src/Game/Characters/Players/BasePlayer.cpp
#include "Game/Characters/Players/BasePlayer.h"

BasePlayer::BasePlayer(std::string name, int hp, int speed, 
                       int hit, int armor, int itemModifier, 
                       int speedModifier, int dodgeModifier, bool enemy) :  
                                BaseCharacter(name, hp, speed, hit, armor, 
                                              itemModifier, speedModifier, 
                                              dodgeModifier, enemy) {}

File: everything_else/ai/finiteStateMachine/combatSimulator/characters/Player/BasePlayer.py
from .. import BaseCharacter

import random

class BasePlayer(BaseCharacter.BaseCharacter):
    def __init__(self, name, hp, speed, hit, armor, itemModifier, speedModifier, dodgeModifier):
        super().__init__(name, hp, speed, hit, armor, itemModifier, speedModifier, dodgeModifier)
        #self.moveSet = [self.attack]
        self.validMovesAndRanges = {0: [1,2], 1:[3,4], 2:[5,6]} #these have to be relative positions
        #(edit: added further ranges to make the combat sim rounds go quicker)

